docs/product/issues/issue-details/replay-issues/rage-clicks.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Rage Click Issues
+sidebar_order: 40
+description: "Learn about Session Replay rage click issues."
+---
+
+If you've enabled [Session Replay](/product/explore/session-replay/), you'll be able to see rage click issues on the [**Issues**](https://sentry.io/orgredirect/organizations/:orgslug/issues/) page in Sentry. Rage clicks are a series of consecutive clicks on the same unresponsive page element. They are a strong signal of user frustration and most likely deserve your attention.
+
+![Rage click issue](./img/Rage-click-issue-in-issue-stream.png)
```

Comment:
Should we take this moment to update this image to an Empower Plant one to avoid having copyright content on the docs?